### PR TITLE
BigQuery: new snippets

### PR DESCRIPTION
**Pull request by shollyman**
Adds new snippets related to imports, view management, and addresses a test flake with table undeletion.

### CODE REVIEW

File: bigquery/snippets/snippet.go [hunk 1]
```diff
@@ -817,6 +845,84 @@ func updateTableExpiration(client *bigquery.Client, datasetID, tableID string) e
...
+	ctx := context.Background()
+	// [START bigquery_update_view_query]
+	view := client.Dataset(datasetID).Table(viewID)
+	meta, err := view.Metadata(ctx)
+	if err != nil {
+		return err
+	}
+
+	newMeta := bigquery.TableMetadataToUpdate{
+		// this example updates a view into the shakespeare dataset to exclude works named after kings.
```
Comment (tbpg) [inline bigquery/snippets/snippet.go]: Capitalize.

File: bigquery/snippets/snippet.go [hunk 2]
```diff
@@ -817,6 +845,84 @@ func updateTableExpiration(client *bigquery.Client, datasetID, tableID string) e
...
+	// that group to query the view, but they must have direct access to any tables referenced by the view.
+	vMeta, err := viewDataset.Metadata(ctx)
+	if err != nil {
+		return err
+	}
+	vUpdateMeta := bigquery.DatasetMetadataToUpdate{
+		Access: append(vMeta.Access, &bigquery.AccessEntry{
+			Role:       bigquery.ReaderRole,
+			EntityType: bigquery.GroupEmailEntity,
+			Entity:     "example-analyst-group@google.com",
```
Comment (tbpg) [inline bigquery/snippets/snippet.go]: Confirming this is the correct email.

File: bigquery/snippets/snippet.go [hunk 3]
```diff
@@ -1468,6 +1587,57 @@ func importCSVExplicitSchema(client *bigquery.Client, datasetID, tableID string)
...
+	if err != nil {
+		return err
+	}
+	status, err := job.Wait(ctx)
+	if err != nil {
+		return err
+	}
+
+	if status.Err() != nil {
+		return fmt.Errorf("Job completed with error: %v", status.Err())
```
Comment (tbpg) [inline bigquery/snippets/snippet.go]: Nit: errors should start with a lower case letter. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

File: bigquery/snippets/snippet.go (hunk 2 above)
Comment (shollyman) [reply bigquery/snippets/snippet.go]: Confirmed.  Alix set this up for the canonical example.

**Pull request merged**